tests/virt_autotest/reboot_and_wait_up.pm

```diff
@@ -42,6 +42,7 @@ sub reboot_and_wait_up {
     else {
         #leave ssh console and switch to sol console
         switch_from_ssh_to_sol_console(reset_console_flag => 'off');
+        send_key 'ret' unless check_screen('text-logged-in-root');
```

Comment:
why should this be necessary? If it's a workaround for some issue then please add a comment to explain that

Reply:
yes, it is neccessary for some SUTs. add the comment to the script:

\#on some SUTs, fg. a HP machine in Beijing lab, screen of sol console is cleared at the second reboot.
\#and the 'return' key must be typed then 'login' prompt appears in sol console